### ISSUE TITLE
What is the initial learning rate for the snap model?

### ISSUE THREAD

**Issue by kishore-greddy** [mattpoggi/mono-uncertainty#10]
Hey @mattpoggi ,

I tried to look for initial learning rate for the "snap" training mode, but I could not find it.  A couple of questions related to snap,
1) The original Snap authors use initial learning rates of 0.2 and 0.1, Did you also use 0.2 as the initial learning rate?
2) You use "C" as 20 meaning one Snap every epoch and choose last N Snaps or N random Snaps? (N being 8 according to your paper), Also T = total number of training iterations , which means for eigen zhou split it would be (39810//batch_size)* num_epochs? 

Thanks in advance 👍

**Comment by mattpoggi** [comment]
Hi @kishore-greddy ,
1. We used the original learning rate from monodepth2
2. We keep the last N snaps. Your definition of T is correct

**Comment by kishore-greddy** [comment]
Thank you
